index.js

```diff
@@ -114,7 +114,13 @@ module.exports = function create (opts) {
         menubar.window.setPosition(x, y)
       }
 
-      if (!opts['always-on-top']) menubar.window.on('blur', hideWindow)
+      if (!opts['always-on-top']) {
+        menubar.window.on('blur', hideWindow)
+      }
+
+      if (opts['show-on-all-workspaces'] !== false) {
```

Comment:
Yes, we should use camelCase. What about the `always-on-top` option? Should we also rename that and make a major release? Or do you like to support both notations for that option?